src/lib/package-make.sh

```diff
@@ -95,6 +95,7 @@ function makepkg-systemd-nspawn() {
   #shellcheck disable=SC2086
   systemd-nspawn -M "${_MECHA_NAME}" \
     -u "root" \
+    --pipe \
```

Comment:
Are you convinced that we want it? As per the manpage:

```
  Input/Output Options
       --console=MODE
          ...snip

           In pipe mode, /dev/console will not exist in the container. This means that the container payload generally cannot be a full init system as init systems tend to require
           /dev/console to be available. On the other hand, in this mode container invocations can be used within shell pipelines. This is because intermediary pseudo TTYs do not permit
           independent bidirectional propagation of the end-of-file (EOF) condition, which is necessary for shell pipelines to work correctly.  Note that the pipe mode should be used
           carefully, as passing arbitrary file descriptors to less trusted container payloads might open up unwanted interfaces for access by the container payload. For example, if a
           passed file descriptor refers to a TTY of some form, APIs such as TIOCSTI may be used to synthesize input that might be used for escaping the container. Hence pipe mode
           should only be used if the payload is sufficiently trusted or when the standard input/output/error output file descriptors are known safe, for example pipes.
```
this implies a security risk, we are building from _unknown sources_ (after a package is added, it isn't re-evaluated again to perform the update), what are the reasons to use it?

Reply:
Anything else doesn't work here sadly, the way it's set up bash is sending it nothing more than EOF right now, which is basically how any non-interactive shell acts I believe? There is no pseudo tty or anything there attached.